libs/openchallenges/home/src/lib/challenge-search/challenge-search.component.html

```diff
@@ -14,14 +14,13 @@ <h1>
         </ngx-typed-js>
       </h1>
       <p class="mat-body-strong">
-        OpenChallenges strives to provide a centralized hub of biomedical challenges from various
-        hosting organizations. Our goal is to empower both participants with the most up-to-date 
-        information about relevant challenges, as well as organizers with standardized challenge
-        templates and intelligence.
+        OpenChallenges is a centralized hub for biomedical challenges and more. It provides
+        participants with the most up-to-date information about relevant challenges, and organizers
+        with standardized challenge event templates and intelligence.
```

Comment:
Overall, it looks more strong intro on the OC 💪. I feel "and organizaers ... and intelligence" is not super clear distinction from the sentence, maybe ?
```suggestion
        OpenChallenges is a centralized hub for biomedical challenges and more. It empowers
        participants with the most up-to-date information about relevant challenges, while providing organizers
        with standardized challenge event templates and intelligence.
```